boards/serpente/board_config.h

```diff
@@ -0,0 +1,17 @@
+#ifndef BOARD_CONFIG_H
+#define BOARD_CONFIG_H
+
+#define CRYSTALLESS    1
+
+#define VENDOR_NAME "arturo182"
+#define PRODUCT_NAME "Serpente"
+#define VOLUME_LABEL "SERPENTBOOT"
+#define INDEX_URL "https://serpente.solder.party/"
+#define BOARD_ID "SAMD21E18A-Serpente"
+
+#define USB_VID 0x239A
+#define USB_PID 0x001E
```

Comment:
hiya please dont re-use trinket m0 PID for this or circuitpython - if you need a PID you can open an issue in `circuitpython` to request one :)

Reply:
Will do, thank you :)